script.js

```diff
@@ -1 +1,186 @@
+  progressBarCount.textContent = `${doneCount} / ${totalCount}`;
+
+  const progressBarDone = document.querySelector(".progress-box__bar-done");
+  let doneRatio = 0;
+  if (totalCount !== 0) doneRatio = (doneCount / totalCount) * 100;
+  progressBarDone.style.width = `${doneRatio}%`;
+
+  // 텍스트 세팅
+  const progressText = document.querySelector(".progress-box__text");
+  if (totalCount !== 0 && doneCount === totalCount) {
+    progressText.textContent = "대단해요 !";
+  } else {
+    progressText.textContent = "달려봐요 !";
+  }
+};
+
+// Todo 또는 Done class 토글 함수
+const toggleTodo = (e) => {
+  const todoListNode = e.target.parentElement;
+  todoListNode.classList.toggle("done");
+  // 이동
+  moveItem(todoListNode);
+  // Progess bar 업데이트
+  updateItemCount();
+  // Local Storage 업데이트
+  setLocalStorage();
+};
+
+// 아이템 삭제 함수
+const deleteItem = (e) => {
+  const todoListNode = e.target.parentElement.parentElement;
+  // 삭제
+  todoListNode.remove();
+
+  const isDone = todoListNode.classList.contains("done");
+  if (isDone) {
+    doneArr = doneArr.filter((element) => element !== todoListNode.textContent);
+  } else {
+    todoArr = todoArr.filter((element) => element !== todoListNode.textContent);
+  }
+
+  // Progess bar 업데이트
+  updateItemCount();
+  // Local Storage 업데이트
+  setLocalStorage();
+};
+
+// Todo/Done 노드 생성 & 계층 세팅 함수
+const createListElement = (txt) => {
+  const todoListNode = document.createElement("li");
+  const todoTextNode = document.createElement("span");
+  const todoDeleteBtnNode = document.createElement("button");
+  todoDeleteBtnNode.innerHTML = '<i class="fa-regular fa-circle-xmark"></i>';
+  todoListNode.append(todoTextNode, todoDeleteBtnNode);
+
+  // Todo 텍스트 설정
+  todoTextNode.textContent = txt;
+  todoTextNode.addEventListener("click", toggleTodo);
+  // Delete 버튼 설정
+  todoDeleteBtnNode.addEventListener("click", deleteItem);
+
+  return todoListNode;
+};
+
+// 할일 추가 함수
+const addTodo = () => {
+  // input에 입력한 값 가져오기
+  const todoInput = document.querySelector(".input-box input");
+  const todoInputText = todoInput.value.trim();
+
+  // 예외처리: 할일 중복 X
+  if ([...todoArr, ...doneArr].includes(todoInputText)) {
+    // 중복 할일 X
+    alert("이미 존재하는 할 일 입니다!");
+    todoInput.value = "";
+    return;
+  }
+
+  // 예외처리: 할일 공백 X
+  if (!todoInputText) return;
+
+  // 추가
+  todoUl.appendChild(createListElement(todoInputText));
+  todoArr.push(todoInputText);
+
+  // Progess bar 업데이트
+  updateItemCount();
+  // Local Storage 업데이트
+  setLocalStorage();
+
+  // input value 초기화
+  todoInput.value = "";
+};
+
+// 첫 화면 렌더링용 함수
+const init = () => {
+  // 오늘 날짜 세팅
+  const date = document.querySelector(".date");
+  date.textContent = getTodayDate();
+
+  // 엔터키 or 플러스 버튼을 누르면 할일 추가
+  const todoInput = document.querySelector(".input-box input");
+  const plusBtn = document.querySelector(".input-box button");
+  todoInput.addEventListener("keyup", function handleEnter(e) {
+    if (e.keyCode === 13) addTodo();
```

Comment:
`keyCode`는 [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) 되어서 `key` 속성을 사용하는 건 어떨까요 ??
```suggestion
    if (e.key=== 'Enter') addTodo();
```